### PR TITLE
Fix move config away from operators

### PR DESCRIPTION
**Pull request by dennisdickmann-digital-power**
moved config parameters for HDXAdm0Operator.py to hdx_adm_dags.py instead of passing and calling config

### CODE REVIEW

File: dags/hdx_adm_dags.py [hunk 1]
```diff
@@ -18,8 +18,15 @@ def create_hdx_adm0_dag(country, schedule_interval, catchup, config, default_arg
     )
     adm0_transform = HDXAdm0Operator(
         task_id=f"hdx_adm0_{country}_transform",
-        country=country,
-        config=config,
+        source='cod',
+        input_filename="/opt/data/test/yem_adm_govyem_cso_ochayemen_20191002_GPKG.zip",
```
Comment (janvdvegt) [inline dags/hdx_adm_dags.py]: These will come from the config later as well, right?

File: plugins/pipeline_plugin/operators/HDXAdm0Operator.py [hunk 2]
```diff
@@ -4,23 +4,23 @@
...
-    print(f"COUNTRY v3.0: {country}")
-    hdx_adm0.transform(source="cod",
-                       input_filename="/opt/data/test/yem_adm_govyem_cso_ochayemen_20191002_GPKG.zip",
-                       schema_filename="/usr/local/airflow/plugins/pipeline_plugin/schemas/admin0_affected_area_py.yml",
-                       output_filename="/opt/data/test/yem_adm0_processed.zip",
-                       country=country,
-                       config=config)
+def transform_adm0(source: str, input_filename, schema_filename, output_filename, iso3, raw_data_dir,
+                   geoboundaries_adm0_raw, schema_mapping, crs, *args, **kwargs):
+    hdx_adm0.transform(source=source, input_filename=input_filename, schema_filename=schema_filename,
```
Comment (janvdvegt) [inline plugins/pipeline_plugin/operators/HDXAdm0Operator.py]: It's cleaner for multiline method calls to have a named argument per line

File: plugins/pipeline_plugin/operators/HDXAdm0Operator.py [hunk 3]
```diff
@@ -4,23 +4,23 @@
...
-            country: str,
-            config,
-            *args, **kwargs) -> None:
-        self.country = country
-        super().__init__(python_callable=transform_adm0, op_kwargs={"country": country, 'config': config}, *args,
-                         **kwargs)
+    def __init__(self, source: str, input_filename, schema_filename, output_filename, iso3, raw_data_dir,
+                 geoboundaries_adm0_raw, schema_mapping, crs, *args, **kwargs) -> None:
+        super().__init__(python_callable=transform_adm0,
+                         op_kwargs={"source": source, "input_filename": input_filename,
```
Comment (janvdvegt) [inline plugins/pipeline_plugin/operators/HDXAdm0Operator.py]: Same here, have a key:value pair per line

File: dags/hdx_adm_dags.py (hunk 1 above)
Comment (dennisdickmann-digital-power) [reply dags/hdx_adm_dags.py]: Yes correct

**Review by janvdvegt — APPROVED** [review]

**Review by cubonacci-demo — APPROVED** [review]

**Pull request merged**